yaml_test.go

```diff
@@ -107,6 +107,7 @@ b:
 }
 
 func unmarshal(t *testing.T, y []byte, s, e interface{}, opts ...JSONOpt) {
+	t.Helper()
```

Comment:
Please remove these t.Helper additions and make this change only about the Go version bump.

I'm not convinced these are proper uses of t.Helper anyway.

Reply:
Ack. I reverted yaml_test.go.